### ISSUE TITLE
`smallbaselineApp.py --plot` fails

### ISSUE THREAD

**Issue by hippopaul** [insarlab/MintPy#878]
I run smallbaselineApp.py --plot to generate plots but it fails

**Full script that generated the error**

```
PASTE CODE HERE
```smallbaselineApp.py 


**Full error message**
Traceback (most recent call last):
  File "/home/bzhang64/.conda/envs/isce2_py38/bin/smallbaselineApp.py", line 8, in <module>
    sys.exit(main())
  File "/home/bzhang64/MintPy/mintpy/cli/smallbaselineApp.py", line 208, in main
    run_smallbaselineApp(inps)
  File "/home/bzhang64/MintPy/mintpy/smallbaselineApp.py", line 1114, in run_smallbaselineApp
    app.open()
  File "/home/bzhang64/MintPy/mintpy/smallbaselineApp.py", line 79, in open
    self._read_template()
  File "/home/bzhang64/MintPy/mintpy/smallbaselineApp.py", line 132, in _read_template
    out_file = os.path.join(backup_dir, os.path.basename(tfile))
  File "/home/bzhang64/.conda/envs/isce2_py38/lib/python3.8/posixpath.py", line 142, in basename
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType

```
PASTE ERROR MESSAGE HERE
```


**System information**

* Operating system: <!--Linux-CentOS,. -->
* Python environment: <!-- conda, -->
* MintPy version: <!-- MintPy version 1.4.1-20, -->
* Your custom / default template file (if the bug is related to a specific dataset): <!-- It helps the diagnose a lot if you could post the configurations you used. You can drag-and-drop them here directly. -->
[template.txt](https://github.com/insarlab/MintPy/files/9862744/template.txt)

**Comment by taliboliver** [comment]
Hi @hippopaul, I haven't look in detail to this error but I noticed that the script goes over two template files, backup and current. In your case one of them is empty. Try making a copy of your config file using a different name and run again. For example if your original config file was "smallbaselineApp.cfg" copy it to something like "myproject.cfg" and then run smallbaselineApp.py myproject.cfg --plot. Alternatively you can plot your data using view.py.

**Comment by hippopaul** [comment]
Hi Talib!

Many thanks. Your suggestion works. 
Cheers,
Paul

**Comment by yunjunz** [comment]
Thank you @hippopaul for the bug reporting. It's now fixed in #879. Cheers!

**Comment by hippopaul** [comment]
Thanks Yunjun!
